### ISSUE TITLE
A11y_mssql for VSCode_Message Window_ScreenReader:Screen reader is announcing the unnecessary information when tabbing to "message" dropdown.

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode-mssql#17197]
## Actual:
Screen reader is announcing the unnecessary information as "Virtual Document  document, document, document, document, frame"  when tabbing to "message" dropdown.

**Similar issue is observed with Jaws**
**Similar issue is also observed in Select top 1000 Results window_when tabbing to results and messages window.**

## Expected:
Screen reader should not announcing the unnecessary information as "Virtual Document  document, document, document, document" when tabbing to "message" dropdown, instead it should only announce the "message' dropdown and its state.

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Jan2022;#mssql for VSCode;#Win32;#FTP;#DesktopApp;#A11ySev3;#AINotInScope;#WCAG1.3.1;#NVDA;#Jaws;#STP;#Element:Dropdown;#Linked:Bug141903;#LinkedBug;#CEACCApproved;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11
Screen Reader: NVDA(2021.3), Jaws(2021.2103.174)

## Repro Steps:

1. Open Visual studio code 
2. Tab till extension button and search for "mssql"
3. Tab till install and move to the SQL Server  button
4. Tab till Add Connection Button and insert the valid credentials
5. Right click on the server/database node you want to run the new query on.
6. Press enter “New Query” write any valid query and click on "Execute query button"
7. Message Window will open
8. Run NVDA and tab to the  "message" dropdown and check if the NVDA is announcing any unnecessary information.

## User Impact:
Screen reader users will get confused if the screen reader is announcing all the unnecessary info

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
## Attachment
 
![17197](https://user-images.githubusercontent.com/86601127/149123217-7969813a-2258-4ae5-8c03-454563bd746a.gif)
![17197 2](https://user-images.githubusercontent.com/86601127/149124793-82560df2-0ad6-4408-a246-3d85250ccbeb.gif)

**Comment by smartguest** [comment]
I agree that the number of technical details about the pane structure is very unnecessary and confusing for users. We will have to investigate whether this bug is happening due to the structure of our vscode-mssql extension or is dependent on the window structure of vscode itself.

**Comment by smartguest** [comment]
The same "virtual document document document" description is encountered when opening a notebook in vscode, and it's not just limited to our extension. Closing as a result

**Comment by Roopesh-T** [comment]
As one of the external bugs is open https://github.com/microsoft/vscode/issues/141903 Please Open this bug and do not close this bug. Only a C & AI tester should close this bug.

**Comment by smartguest** [comment]
"Virtual Document" is no longer being said as the title has been removed from vscode, but it still announces document, document when tabbing to the results tab from another place, this means the structure of our extension/document needs to be labeled differently to account for this.

**Comment by smartguest** [comment]
Issue still occurs for our extension specifically, although Virtual Document is no longer announced.

It appears every time you create a new HTML sub window in vscode (perhaps only in an iframe like our extension), it applies a document attribute to the html section automatically. 

Screenshot showing Document attributes and iframe class.
![Iframedoctype html](https://user-images.githubusercontent.com/18018452/174195887-c3e5a7ea-5db0-494d-9e6c-39cb08cdcafe.png)

Removing the role="document" from the body section didn't seem to do anything for the screen reader and this is likely applied by vscode (none of our code mentions "document" in regard to generating the window.)

Below screenshots showing 'role = "document"' in body section(s) 

![Iframedoctype bodyrole1](https://user-images.githubusercontent.com/18018452/174196154-879fdf0c-e4b1-4d00-9185-017735fe9150.png)

![Iframedoctype bodyrole2](https://user-images.githubusercontent.com/18018452/174196157-fad6e4df-7d86-4613-a660-000b731c4a42.png)

**Comment by smartguest** [comment]
@Roopesh-T Issue is still being investigated by the vscode team currently, although they may not be able to find a fix if it's part of an external dependency in their code.

**Comment by Roopesh-T** [comment]
Reducing the Sevirity of the bug to Sev3 as per C&AI SEM's discussion.

**Comment by fsteffi** [comment]
GithubTags:#External;#External:VSCode;

**Comment by keerthiduvvuri** [comment]
GithubTags:#ResolvedByTT

**Comment by aasimkhan30** [comment]
Doesn't happen on new results grid.

<img width="517" height="35" alt="Image" src="https://github.com/user-attachments/assets/413c75a5-f1cc-44ab-9b3b-cc1e4444acbb" />
